app/components/candidate_interface/degrees_review_component.rb

```diff
@@ -109,7 +114,8 @@ def enic_reference_row(degree)
         key: t('application_form.degree.enic_reference.review_label'),
         value: degree.enic_reference,
         action: generate_action(degree: degree, attribute: t('application_form.degree.enic_reference.change_action')),
-        change_path: Rails.application.routes.url_helpers.candidate_interface_edit_degree_enic_path(degree.id),
+        change_path: candidate_interface_edit_degree_enic_path(degree.id, return_to_params),
+        data_qa: 'degree-enic-reference',
```

Comment:
This page also needs the return to params added 

![image](https://user-images.githubusercontent.com/42515961/128697072-d4f7e3fa-730c-45a8-9a53-a9ead71f5853.png)